packages/foreman/nodejs-hoist-non-react-statics/nodejs-hoist-non-react-statics.spec

```diff
@@ -1,16 +1,24 @@
+%{?scl:%scl_package nodejs-%{npm_name}}
+%{!?scl:%global pkg_name %{name}}
+
 %global npm_name hoist-non-react-statics
 
-Name: nodejs-hoist-non-react-statics
+Name: %{?scl_prefix}nodejs-hoist-non-react-statics
 Version: 2.5.5
-Release: 1%{?dist}
+Release: 2%{?dist}
 Summary: Copies non-react specific statics from a child component to a parent component
 License: BSD-3-Clause
 Group: Development/Libraries
 URL: https://github.com/mridgway/hoist-non-react-statics#readme
 Source0: https://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
+%if 0%{?scl:1}
+BuildRequires: %{?scl_prefix_nodejs}npm
```

Comment:
This NPM is still weird. It shouldn't be needed for single (non-bundled) packages, right?

Reply:
This is already in half the packages we updated. I'd suggest we keep it and come back through and make updates as needed